Tests/RockLib.Messaging.CloudEvents.Tests/CloudEventExtensionsTests.cs

```diff
@@ -132,7 +133,7 @@ public void SetDataMethod2SadPath()
         public void SetDataMethod3HappyPath1()
         {
             var cloudEvent = new CloudEvent();
-            var client = new Client { FirstName = "Brian", LastName = "Friesen" };
+            var client = new TestClient { FirstName = "Brian", LastName = "Friesen" };
```

Comment:
We should clear the bad mojo

```suggestion
            var client = new TestClient { FirstName = "Neil", LastName = "Schiefer" };
```

Reply:
updated to use generic names.